File: uri/media/media.go
package main

import(
	"C"
	"fmt"
)

var peso1 float64 = 3.5
var peso2 float64 = 7.5

func media( nota1 float64, nota2 float64 ) float64 {
	return ( (nota1 * peso1) + (nota2 * peso2)   ) / 11.0
}

func main(){
	var nota1, nota2 float64
	fmt.Scanf("%f\n", &nota1)
	fmt.Scanf("%f\n", &nota2)
	fmt.Printf("MEDIA = %.5f\n",  media(nota1, nota2))
}

File: uri/media/media.js
//var input = require('fs').readFileSync('stdin.txt', 'utf8');
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var peso1 = 3.5;
var peso2 = 7.5;

function media( nota1, nota2 ){
    return ((( nota1 * peso1) + (nota2 * peso2)) / 11.0);
}

var p1 = parseFloat( lines.shift() );
var p2 = parseFloat( lines.shift() );
var med = media( p1, p2 );
console.log('MEDIA = ' + med.toFixed(5));